src/ai/backend/manager/registry.py

```diff
@@ -1391,51 +1435,14 @@ async def _post_create_kernel(
             log.exception("post_create_kernel(k:{}) unexpected error", kernel_id)
             return
         else:
+            await self.finalize_running(
```

Comment:
Here it should also use `execute_with_retry()`.

Reply:
`finalize_running` method use `execute_with_retry()` internally, so I do not wrap this.